src/pages/Delivery/components/DeliveryModal/DeliveryModal.jsx

```diff
@@ -0,0 +1,182 @@
+import { Stack } from "@mui/system";
+import { useState } from "react";
+import AutocompleteBase from "../../../../components/AutocompleteBase/AutocompleteBase";
+import ModalBase from "../../../../components/ModalBase/ModalBase";
+import Input from "../../../../components/Search/components/Input/Input";
+import TextFieldBase from "../../../../components/TextFieldBase/TextFeldBase";
+import SearchIcon from "@mui/icons-material/Search";
+import { InputAdornment } from "@mui/material";
+
+const stackStyles = {
+  display: "flex",
+  flexFlow: "row wrap",
+  gap: "1rem",
+  mt: "32px",
+  justifyContent: "center",
+};
+
+const DeliveryModal = () => {
+  return (
+    <>
+      <ModalBase
+        titleModal="New Delivery"
+        descriptionModal="Please select the order ID and a technician to deploy the cargo. All
+            elements are mandatory."
+        renderBody={() => {
+          return (
+            <>
+              <Stack sx={stackStyles}>
+                <TextFieldBase labelProp="Order Id" />
+                <AutocompleteBase
+                  startAdornment={
+                    <InputAdornment position="start">
+                      <SearchIcon
+                        sx={{ ml: "1rem", color: "rgba(0, 0, 0, 0.25)" }}
+                      />
+                    </InputAdornment>
+                  }
+                  options={top100Films}
+                  labelProp="Technician"
+                />
+              </Stack>
+              <Stack sx={stackStyles}>
+                <AutocompleteBase options={top100Films} labelProp="Platform" />
+                <AutocompleteBase options={top100Films} labelProp="Drone" />
+              </Stack>
+            </>
+          );
+        }}
+      />
+    </>
+  );
+};
+
+export default DeliveryModal;
+
```

Comment:
maybe you could place this array into another file to not bloat this component

Reply:
Good I'll do it